frontend/src/pages/Organization/Organization.tsx

```diff
@@ -656,6 +657,7 @@ export const Organization: React.FC = () => {
           />
           <TextField
             margin="dense"
+            inputProps={{ maxLength: 20 }}
```

Comment:
@ValeriaReveles can you please remove these maxLength changes from this PR, so it only has the styles.css change? Thank you!

Reply:
@ValeriaReveles Can you please make these changes? Thanks!

Reply:
Changes made, new PR submitted.